src/domain/event/event.module.scss

```diff
@@ -110,3 +110,38 @@
     }
   }
 }
+
+.occurrenceInfo {
+  margin: $baseMargin 0;
+  width: 67%;
+  display: block;
+  .label {
+    display: inline-flex;
+    width: min-content;
+    @include respond-above(sm) {
+      > * {
+        margin-left: 0;
+      }
+    }
+  }
+  & > * {
+    margin-left: 0;
+    margin-right: $baseMargin;
+    white-space: nowrap;
+  }
+}
```

Comment:
An alternative could be to create a small `<Label />` (icon + text) component since they are in freely ordered and in many place

Reply:
Good idea. I propose that we do that in a follow up PR